internal/core/core_test.go

```diff
@@ -18,9 +18,10 @@ func make_connection(name string) Connection {
 	return NewFileConnection(path)
 }
 
-var well_known = make_connection("well_known")
-var samples10 = make_connection("10_samples")
-var prestack = make_connection("prestack")
+var   well_known = make_connection("well_known")
+var   samples10  = make_connection("10_samples")
+var   prestack   = make_connection("prestack")
+const fillValue  = float32(-999.25)
```

Comment:
I'm not a fan of formatting unrelated code like this. I highly encourage you do to these kinds of small changes when you see them, but always as separate commits

Reply:
Sure